tests/test_event_auth.py

```diff
@@ -37,21 +37,19 @@ def test_random_users_cannot_send_state_before_first_pl(self):
         }
 
         # creator should be able to send state
-        event_auth.check(
+        event_auth.check_auth_rules_for_event(
```

Comment:
since all these tests are just trying to check that the auth rules are checked, we may as well skip straight to that bit.